packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -119,21 +112,21 @@ export enum Language {
   chinese = 'zh-cn',
 }
 
-export const localizationGuard = z.object({
-  autoDetect: z.boolean(),
-  primaryLanguage: z.nativeEnum(Language),
-  fallbackLanguage: z.nativeEnum(Language),
+export enum LanguageMode {
+  auto = 'auto',
+  fixed = 'fixed',
+}
+
+export const languageInfoGuard = z.object({
+  mode: z.nativeEnum(LanguageMode),
+  fallback: z.nativeEnum(Language),
```

Comment:
should we keep 'primaryLanguage' or fallback will come to use while the mode is set to be fixed?

Reply:
split `fallback` into `fallbackLanguage` and `fixedLanguage`. PTAL